api/samples/CMakeLists.txt

```diff
@@ -122,6 +122,13 @@ else (WIN32)
   set(OPT_CFLAGS "-O2")
 endif (WIN32)
 
+option(NO_TRACE_OUTPUT
```

Comment:
It's not related to the vector length. This was added to avoid unmanageably large trace files when testing the sample clients with non-trivial SVE applications and benchmarks. Thinking about it now, it may be best to remove `NO_TRACE_OUTPUT` and find a better, less hard-wired way of dealing with large traces in a separate patch. Suggestions/ideas welcome.

Reply:
The samples are deliberately simple.  If drmemtrace tests the same functionality then it could be used to test instead as it has various existing options to cap the output size.

Reply:
Ok, we will use drmemtrace to test. We may raise a separate issue and PR to suppress trace output in the sample clients dynamically for test purposes, perhaps as client CLI option.